applications/office-365/teams-chat.js

```diff
@@ -38,11 +38,8 @@ async function runScript() {
   // This click loads the main teams page
   await markerClick(By.id('idBtn_Back'), "Login", "Backend Load");
 
-  // Dismiss the "use desktop version" message
-  await clickPeriodic(By.css(`[data-tid="early-desktop-promo-use-web"]`), 500);
-
   // Wait for chat page to load
-  await driver.wait(until.elementIsVisible(await driver.findElement(By.css(`.team-information`))))
+  await driver.wait(until.elementIsVisible(await driver.findElement(By.css(`.team-information`))));
```

Comment:
@quig-te - make sure to add explicit timeout to all `driver.wait` calls so that tests don't hit generic timeout.

Reply:
OK please review....not sure if I pushed the commit to the PR appropriately.  Do you see the additional Changes?